dashboard/test/models/concerns/user_permission_grantee_test.rb

```diff
@@ -123,6 +123,15 @@ class UserPermissionGranteeTest < ActiveSupport::TestCase
     assert user.workshop_organizer?
   end
 
+  test 'ai_chat_access?' do
+    user = create :teacher
+    refute user.ai_chat_access?
+    user.permission = UserPermission::LEVELBUILDER
```

Comment:
Similar question as above, is there something about levelbuilder permissions you're trying to test?

Reply:
No - just getting a feel for writing Ruby tests. Thanks!